src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sumAllNumers = 0;
+
+  const adder = (numbers) => {
```

Comment:
it's just a single number, because numbers it's like an array or smth
```suggestion
  const adder = (number) => {
```